implementation.md

```diff
@@ -200,6 +200,10 @@ the `Input` instruction. This class must be annotated with
 1. Create new classes that provide data which implement _R_ and are annotated
 with `@MetaInfServices(`_R_`)`.
 
+For each variable, Shesmu can try to infer the type from the return type of the
+method. If this is not possible, it must be specified in the `type` property of
```

Comment:
Type can be inferred for boolean, long, Instant, and String, but Set and Tuple types must be specified, right? Please note that here

Reply:
Fixed. Thanks.